src/main/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/service/LicenceConditionsService.kt

```diff
@@ -8,6 +8,7 @@ import uk.gov.justice.digital.hmpps.makerecalldecisionapi.domain.makerecalldecis
 import uk.gov.justice.digital.hmpps.makerecalldecisionapi.domain.makerecalldecisions.LicenceConditionsResponse
 import uk.gov.justice.digital.hmpps.makerecalldecisionapi.domain.makerecalldecisions.Offence
 import uk.gov.justice.digital.hmpps.makerecalldecisionapi.domain.ndelius.ReleaseSummaryResponse
+import kotlin.streams.toList
 
```

Comment:
Do we need this import? There are no other changes to that file

Reply:
My IDE complains and errors when it is not there :(